frontend/src/component/project/Project/ProjectSettings/ProjectSettings.tsx

```diff
@@ -35,11 +44,11 @@ export const ProjectSettings = () => {
             onChange={onChange}
         >
             <Routes>
-                <Route path={`${tabs[0].id}/*`} element={<ProjectAccess />} />
                 <Route
-                    path={`${tabs[1].id}/*`}
+                    path={`${tabs[0].id}/*`}
```

Comment:
This seems clearer to me 🤷‍♂️ 
```suggestion
                    path="environments/*"
```